PatientWebApplication/HealthClinic.DAL/Service/RegularAppointmentService.cs

```diff
@@ -82,6 +85,12 @@ public List<DoctorAppointment> GetAppointmentsForPatient(int id)
             return _appointmentRepository.GetAppointmentsForPatient(id);
         }
 
+        public List<DoctorAppointment> GetAppointmentsForDoctor(int id)
+        {
+            List<DoctorAppointment> appointments = _appointmentRepository.GetAppointmentsForDoctor(id);
+            return appointments;
```

Comment:
Write this in form:
"return _appointmentRepository.GetAppointmentsForDoctor(id)"